tests/functional/ext/typing/typing_consider_using_alias.py

```diff
@@ -28,7 +28,6 @@
 var7: typing.Hashable  # [consider-using-alias]
 var8: typing.ContextManager[str]  # [consider-using-alias]
 var9: typing.Pattern[str]  # [consider-using-alias]
-var10: typing.re.Match[str]  # [consider-using-alias]
```

Comment:
It didn't seem worth it to extract this into its own test file that runs on 3.12 and below only. Let's conserve energy.